ios/RNSScreenStack.mm

```diff
@@ -57,6 +57,31 @@ - (UIViewController *)childViewControllerForStatusBarHidden
   return [self topViewController];
 }
 
+- (void)viewDidLayoutSubviews
+{
+  if ([self.topViewController isKindOfClass:[RNSScreen class]]) {
```

Comment:
Shouldn't we call `[super viewDidLayoutSubviews]` here or is it not necessary?

Reply:
It's not necessary, everything works normally without the `super` (Xcode also doesn't complain about it) but I can add one 👍

Reply:
Added one in https://github.com/software-mansion/react-native-screens/pull/1802/commits/9557ec64ce6703d03bf0566d0b96fa2741b6ea18